pages/glossary/changes-in-content.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Changes in content
+key: changes-in-content
+unambiguous: true
+objective: true
+input_aspects:
+  - Accessibility tree
+  - Audio output
+  - CSS styling
+  - DOM tree
+---
+
+A _change in the content_ of a [web page][] caused by an event occurs when, by comparing the [web page][] before and after the event firing, at least one of the following occurs:
+
+- **visible changes:** the rendered pixels change in any part of the document that is currently within the [viewport][] or that can be brought into the [viewport][] via scrolling; or
+- **accessibility tree changes:** any state, property or event of a node representing an [accessible object][] of the [accessibility tree][] changes, or any node is inserted in, or removed from the [accessibility tree][]; or
+- **audible changes:** the audio reproduced by the [web page][] changes.
+
+If the [web page][] is reproducing time-based media, rendered pixels and reproduced audio will be changing as part of the playback. The comparison in this instance should compare the pixels and audio that are rendered and reproduced if the event is not fired, with the ones that are rendered if the event is fired.
+
+For the purpose of this definition, it is assumed that the event causing the changes is the only origin of changes to the [web page][] and therefore the page does not change if there is no such event.
+
+For the purpose of this definition, it is assumed that the changes happen instantly after the event firing and therefore the comparison between the page before and after the event firing can be made at any time after the firing of the event.
```

Comment:
I think the assumptions:
1. should be made more proeminent, e.g. with an "Assumptions" section in the def.
2. should be copied in any rule using that definition, given that if the definition gives bad result, then the rule will also give bad result (probably together with a "conclusion" like "if this does not hold, then the def will produce nonsense and the rule will fail because of that without the SC failing" as we usually have.

(and I think that doing 1. also helps remembering to do 2.)

We don't really have assumptions in definitions so far, so there is no real agreement on how to handle them and no guidelines, so this is mostly personal opinion on how to handle them…

Reply:
I like the suggestion. Going to implement it :-)